scripts/qa_crowbarsetup.sh

```diff
@@ -3173,6 +3173,18 @@ function set_noproxyvar
     no_proxy="${no_proxy%,}";
 }
 
+# commit a proposal, but use crowbarctl from cloud6 on
+function crowbar_proposal_commit
+{
+    local proposal="$1"
+    local proposaltype="${2:-default}"
+    if iscloudver 6plus ; then
```

Comment:
Good point, the updates are not out yet (they go together with the upgrade update). Should we set the timeout env variable to also cover the GM6 & GM7 case?